.rubocop.yml

```diff
@@ -20,7 +20,7 @@ Layout/HashAlignment:
   Enabled: false
 
 Layout/LineLength:
-  Max: 80 # default: 120
+  Max: 80  # default: 120
```

Comment:
requiring 2 spaces before the commend is kinda weird

Reply:
Yeah, I’ll remove that rule.